data/404/227/381/404227381.geojson

```diff
@@ -107,7 +113,7 @@
         "\u0392\u03b1\u03bb\u03ad\u03bd\u03b8\u03b9\u03b1\u03bd\u03b9\u03ba\u03ae \u03ba\u03bf\u03b9\u03bd\u03cc\u03c4\u03b7\u03c4\u03b1"
     ],
     "name:eng_x_preferred":[
-        "Valencian Community"
+        "Valencia"
```

Comment:
Hmm now this value doesn't match the wof:name.. we also lose the existing name..

Reply:
In this case, awhile ago we had decided thru manual curration the longer `Valencian Community` was the preferred English name and `wof:name` to differentiate it from the locality of the same name?

So should I preserve the existing preferred English name in Spain, and the default wof:name for macroregions?

And generally I should store edited names as variants...

Reply:
This only affected 3 `macroregions` in Spain. I'll restore the names to their previous curated values.

There are 245 total records where the preferred English name is changing, including 83 records in Taiwan where they were actually Chinese characters instead of ASCII-7 values. In both cases I'll look to see if the value is currently in English variants and add if not, or Chinese primary or variants and add as variant if not.

Reply:
Just so it's all in one spot...

This also happens for `label:eng_x_preferred_placetype` for 1696 records across all admin repos, but except for 11 records exclusively in Russia (majority) and Sudan (minority). I'll store the earlier value in `label:eng_x_variant_placetype`, which I don't think we have used before but is a legal property name.

This also happens for `wof:name` in 150 records, and 105 of them are Chinese, handful random languages, rest English. I'll look in current preferred and variant names for those languages (manual review) and add if appropriate.

Reply:
This is fixed via new commit, but the DIFF and comment here looks funny as it's attached to the variant (which is still short deliberately) versus the preferred name.

Reply:
Gotcha. I do see "Valencian Community" is the preferred name, "Valencia" is the variant name. https://raw.githubusercontent.com/whosonfirst-data/whosonfirst-data-admin-es/nvkelso/2154-placetype-local-population-statoids/data/404/227/381/404227381.geojson

In general, I think overwriting names should be avoided (we've avoided that here).. unless the existing name is clearly bogus or wrong. I'm less concerned with label values, but probably better to preserve in variant spaces when needed.

tlrd this looks good now, can do another pass at this if needed.